### PR TITLE
Handle 403 responses on caches page

### PR DESCRIPTION
**Pull request by rafael-travis**
Complements https://github.com/travis-ci/travis-api/pull/929

### CODE REVIEW

**Comment by rafael-travis** [comment]
@artursmirnov thanks a lot for the documentation reference. Is this how you thought about implementing this?

**Comment by artursmirnov** [comment]
Hey @rafael-travis, yes, almost there :) Although, I think that `error` object should be available in the error state context and thus in `error` template (but I'm not sure about this). If it's available, then you can use it directly to display different messages:
```hbs
{{#if (eq error.status 403)}}
  Forbidden error
{{else}}
  Other error
{{/if}}
```
In this case you won't need to define a controller and even catch an `error` event, it should just work.

If `error` object is not available in the `error` substate template directly, I would suggest to add a `repo/error` route and set error object manually, like this: 
```js
setupController(controller, error) {
  this._super(...arguments);
  controller.set('error' error);
}
```
And then use it in template as above.

The issue with your current solution is that controllers in Ember are singletons, so when you set `isForbidden` to it, it will persist there forever, unless you reset it manually.

**Comment by rafael-travis** [comment]
@artursmirnov thanks, I couldn't find a way to do that #if on my researches, that's much better, let me know if this looks good to you.

File: app/templates/repo/error.hbs [hunk 1]
```diff
@@ -1 +1,5 @@
-There was an error while loading data.
+{{#if (eq model.status 403)}}
```
Comment (rafael-travis) [inline app/templates/repo/error.hbs]: @artursmirnov it's accessible through this `model` variable.

**Comment by artursmirnov** [comment]
Hey @rafael-travis, thanks for working on this! It looks great to me 👍 I really like such a straight forward solution. Please merge the PR as soon as you feel it's ready for release to production.

**Comment by rafael-travis** [comment]
@artursmirnov thank you for educating me on Ember, I appreciate it 👍

**Comment by rafael-travis** [comment]
@artursmirnov `percy/travis-web` failed but I don't have access to see what happened, do I need to be added to the team in there?

**Comment by artursmirnov** [comment]
@rafael-travis, no worries, fixed ;)

**Comment by rafael-travis** [comment]
@artursmirnov thanks!

**Pull request merged**